isazonov/src/main/java/com/evilcorp/hcf/functions/HcfContributor.java

```diff
@@ -0,0 +1,31 @@
+package com.evilcorp.hcf.functions;
+
+import org.hibernate.boot.model.FunctionContributions;
+import org.hibernate.boot.model.FunctionContributor;
+
+public class HcfContributor implements FunctionContributor {
```

Comment:
Add javadoc that describes what this class is actually doing

Reply:
Javadoc added